### PR TITLE
docs/variants/talos_2/: update documentation

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki [kamil.pokornicki@3mdeb.com]

### CODE REVIEW

File: docs/variants/talos_2/hardware-matrix.md [hunk 1]
```diff
@@ -5,23 +5,20 @@
...
-|                                | 2. TBD                                                        |
-| **Power Supply**               | TBD W ATX type power supply                                   |
+| **Attached devices**           | 1. SanDisk USB 3.2Gen1 16 GB                                  |
+| **Ethernet**                   | 2x Broadcom BCM5719                                           |
+| **Power Supply**               | Corsair TX550M                                                |
 | **Power Control**              | 1. Sonoff S20 switch                                          |
-|                                | 2. Goldpin cables (RTE <-> Board connection)                  |
-| **Remote Testing Environment** | 1. RTE `v1.0.0` (firmware `v0.5.3`) connected via RS232       |
-|                                | 2. Goldpin cables + qspimux (RTE <-> flash memory connection) |
+|                                | 2. OpenBMC                                                    |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/hardware-matrix.md:21]: What type of the component is OBMC?

File: docs/variants/talos_2/initial-deployment.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Insurgo Dasharo on Talos II - installation manual
+# Insurgo Dasharo on Talos II - initial deployment
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/initial-deployment.md:1]: Take a look on the ASUS or Protectli or Optiplex documentation - we should keep naming headings.

File: docs/variants/talos_2/overview.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Overview
+
+* [Releases](releases.md) - list of all releases
+* [Building manual](building-manual.md) - how to build Dasharo compatible with
+  the Talos II
+* [Initial deployment](initial-deployment.md) - how to install Dasharo
+  firmware on the Talos II
+* [Hardware configuration matrix](hardware-matrix.md) - definition of hardware
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/overview.md:8]: `Firmware update` and `Recovery` sections are missing

File: mkdocs.yml [hunk 4]
```diff
@@ -150,10 +151,10 @@ nav:
 #     - 'Community Edition': inquiries/community-edition.md
 #     - 'Sponsorware': inquiries/sponsorware.md
 #     - 'Commercial inquiries': inquiries/commercial-inquiries.md
-  - 'Dasharo Tools Suite': common-coreboot-docs/dasharo_tools_suite.md
   - 'Knowledge base':
       - 'Common coreboot documentation':
           - 'Vboot signing': common-coreboot-docs/vboot_signing.md
+          - 'Dasharo Tools Suite': common-coreboot-docs/dasharo_tools_suite.md
```
Comment (sulewskiprzemyslaw) [inline mkdocs.yml:157]: Is this change intentional?

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/talos_2/initial-deployment.md (hunk 2 above)
Comment (macpijan) [reply docs/variants/talos_2/initial-deployment.md:1]: I'd take a look at Protectli VP4, as it was the latest release: https://docs.dasharo.com/variants/protectli_vp46xx/initial-deployment/

There is no point in repeating platform name in each document title. Just `Initial deplyment` should suffice.

File: docs/variants/talos_2/hardware-matrix.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/variants/talos_2/hardware-matrix.md:21]: I think that remote control and update it

File: docs/variants/talos_2/initial-deployment.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/variants/talos_2/initial-deployment.md:1]: Sure, I renamed it

File: mkdocs.yml (hunk 4 above)
Comment (Pokisiekk) [reply mkdocs.yml:157]: No, sorry, I restored it

File: docs/variants/talos_2/overview.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/variants/talos_2/overview.md:12]: Sections added

File: docs/variants/talos_2/hardware-matrix.md [hunk 5]
```diff
@@ -5,23 +5,20 @@
...
 
 | Component                      | Description                                                   |
 |--------------------------------|---------------------------------------------------------------|
-| **CPU**                        | TBD                                                           |
-|                                | TBD                                                           |
+| **CPU**                        | IBM POWER9 “Sforza”                                           |
 |                                | CPU Cooler                                                    |
-| **RAM**                        | TBD                                                           |
-| **Flash memory**               | TBD                                                           |
+| **RAM**                        | Crucial CT8G4RF88266                                          |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/hardware-matrix.md:14]: Is this the only one RAM module mounted in the platform?

File: docs/variants/talos_2/other-manuals.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Insurgo Dasharo on Talos II - Conference materails
+# Other manuals
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/other-manuals.md:1]: We should roll back the name of this file - `other manuals` is a collective term.

File: docs/variants/talos_2/recovery.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+Backup files should be generated before making any changes in the device flash
+chip according to documentation in the second step in
+[coreboot installation](initial-deployment.md#coreboot-installation)
+section.
+
+---
+
+## Flashing using BMC
+
+Flash firmware by executing the following commands on BMC:
```
Comment (sulewskiprzemyslaw) [inline docs/variants/talos_2/recovery.md:19]: If after flashing the platform we should not do something more, e.g. reboot the platform?

File: mkdocs.yml [hunk 8]
```diff
@@ -47,12 +47,15 @@ nav:
...
           - 'Building manual': variants/talos_2/building-manual.md
-          - 'Installation manual': variants/talos_2/installation-manual.md
-          #- 'Hardware Configuration Matrix': variants/talos_2/hardware-matrix.md
+          - 'Initial deployment': variants/talos_2/initial-deployment.md
+          - 'Firmware update': variants/talos_2/firmware-update.md
+          - 'Recovery': variants/talos_2/recovery.md
+          - 'Hardware Configuration Matrix': variants/talos_2/hardware-matrix.md
           - 'Test matrix': variants/talos_2/test-matrix.md
-          - 'Conference materials': variants/talos_2/conferences.md
+          - 'Other manuals': variants/talos_2/other-manuals.md
```
Comment (sulewskiprzemyslaw) [inline mkdocs.yml:58]: We should roll back the name of this file - `other manuals` is a collective term.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/talos_2/hardware-matrix.md (hunk 5 above)
Comment (Pokisiekk) [reply docs/variants/talos_2/hardware-matrix.md:14]: Currently yes

File: docs/variants/talos_2/other-manuals.md (hunk 6 above)
Comment (Pokisiekk) [reply docs/variants/talos_2/other-manuals.md:1]: renamed

File: mkdocs.yml (hunk 8 above)
Comment (Pokisiekk) [reply mkdocs.yml:58]: renamed

File: docs/variants/talos_2/recovery.md (hunk 7 above)
Comment (Pokisiekk) [reply docs/variants/talos_2/recovery.md:19]: I added missing steps

**Review by sulewskiprzemyslaw — APPROVED** [review]
